components/director/internal/config/config.go

```diff
@@ -0,0 +1,6 @@
+package config
+
+type ExternalSystemConfig struct {
+	InternalConnectorURL string `envconfig:"APP_INTERNAL_CONNECTOR_URL"`
```

Comment:
existence of `InternalConnectorURL` and `ExternalConnectorURL` is not obvious, please add description of why we need it